Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY main.go .
 COPY database database
 COPY logging logging
 COPY sse sse
+RUN go get mvdan.cc/xurls/v2
```

Comment:
why do this in the dockerfile? wouldn't it be better to have this as part of the go.mod and go.sum?